c/src/ml-api-service-remote.c

```diff
@@ -545,16 +565,17 @@ ml_service_remote_create (ml_option_h option, ml_service_h * handle)
 
   _mlrs_get_edge_info (option, edge_info);
 
-  ret = _mlrs_create_edge_handle (&edge_h, edge_info);
+  remote_s = g_new0 (_ml_remote_service_s, 1);
```

Comment:
Is it ok with not to free it in the error cases?

Reply:
Changed to free it in the error case. Thanks.